modules/virt-creating-vddk-image.adoc

```diff
@@ -47,16 +44,18 @@ EOF
 . Build the image:
 +
 ----
-$ docker build -f Dockerfile -t <server/username/reponame:tag> <1>
+$ podman build . -t <server_name>/path/to/vddk:<tag> <1>
 ----
-<1> Specify your repository details.
+<1> Specify the image registry, path, and tag:
++
+* For an external registry, specify the server name, for example, `server.example.com:5000/vddk:v670`.
+* For the internal registry, use the internal registry route, for example, `image-registry.openshift-image-registry.svc:5000/openshift/vddk:<tag>`.
```

Comment:
The first bullet uses `vddk:v670` and this second bullet uses `vddk:<tag>`. Just wondering whether they should be consistent between the two or not

Reply:
No reason not to make them consistent. Thanks